sql/core/src/main/scala/org/apache/spark/sql/execution/window/WindowExec.scala

```diff
@@ -197,6 +198,7 @@ case class WindowExec(
 
     // Map the groups to a (unbound) expression and frame factory pair.
     var numExpressions = 0
+    val timeZone = conf.sessionLocalTimeZone
```

Comment:
NIT: should we also touch up the use of `sqlContext.conf` in `doExecute()`?

Reply:
that one is OK because it's not in a closure and passed to executors.